pepper-apis/docs/specification/src/endpoints/user.studies.activity.yml

```diff
@@ -41,3 +41,37 @@ get:
       $ref: '../pepper.yml#/components/responses/ActivityDetailNotFoundResponse'
     default:
       $ref: '../pepper.yml#/components/responses/ErrorResponse'
+patch:
+  operationId: patchActivity
+  tags:
+    - Activities
+  summary: updates section index
+  description: with given user Guid, study's unique identifier and instance Guid Updates user's section index
```

Comment:
Can you make these a bit more generic? E.g. `summary` should be something `update activity instance metadata`, and `description` should say something about updating properties of activity instance but that currently only section index is supported.